vm/compiler/src/ledger/get.rs

```diff
@@ -75,6 +75,15 @@ impl<N: Network, B: BlockStorage<N>, P: ProgramStorage<N>> Ledger<N, B, P> {
         }
     }
 
+    /// Returns the transaction for the given transaction id.
+    pub fn get_transaction(&self, transaction_id: &N::TransactionID) -> Result<Transaction<N>> {
```

Comment:
```suggestion
    pub fn get_transaction(&self, transaction_id: N::TransactionID) -> Result<Transaction<N>> {
```

Do not pass by reference. Since N::TransactionID implements Copy

Reply:
Whoops, you're right; done